src/common/xo/index.js

```diff
@@ -1026,6 +1026,10 @@ export const deleteVms = vms =>
   confirm({
     title: _('deleteVmsModalTitle', { vms: vms.length }),
     body: _('deleteVmsModalMessage', { vms: vms.length }),
+    strongConfirm: vms.length > 10 && {
```

Comment:
IMHO, it should applies to any number of VMs.

Reply:
Why? Wouldn't it be overkill to ask the user to type a message in order to delete 1 or 2 VMs?

Reply:
I think it makes sense when you remove more VMs than displayed in the current page.

Reply:
I think we should be conservative for the moment and ask for strong confirmation for any number of VMs for the moment.

We'll relax this later if necessary based on the users feedback :)

Reply:
IMHO, as a user, I would hate having to do it everytime. We shouldn't destroy the user experience for doing experiments: this is what we should keep in mind by being careful.

Reply:
This is only when removing multiple VMs from the home view, I don't think that's a very frequent (multiple times a day) operation.

If users complain about it we can make release a fix in a couple of minutes anyway.

Reply:
I am doing that quite a lot :D

Reply:
So I pre-complain here :o

Reply:
Noted 😉

Reply:
I'll test it anyway, because I didn't so far. Maybe it's not that hard to type it everytime. Let's see.